diesel/src/query_dsl/distinct_dsl.rs

```diff
@@ -11,10 +11,10 @@ use expression::SelectableExpression;
 ///
 /// [`QueryDsl`]: ../trait.QueryDsl.html
 pub trait DistinctDsl {
-    /// Query with DISTINCT added
+    /// The type returned by `.distinct`
     type Output;
 
-    /// Adds `DISTINCT` to to the query.
+    /// See the trait documentation.
```

Comment:
This is pretty non-documentary. If this is what you really want people to do, at least include a link

Reply:
I'd be linking to the page the user is currently on. There are plenty of examples in libstd where a module that basically only has one trait puts all its documentation in the module docs, and then has the trait documented as "see the module docs". I figured the same applies reasonably well to traits with only one method (in general the goal here being to have the docs render as high up the page as possible).

TL;DR: Given that the trait only has one method, and the trait docs are written as though they were the docs for this method, I think this is fine. The only real alternative is moving the trait docs to this method, and linking to them in the trait docs, but the trait docs render higher up on the page.